File: 백준/Bronze/14541. Speed Limit/Speed Limit.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        while (true) {
            int n = sc.nextInt();
            if (n == -1) {
                break;
            }
            
            int dap = 0, f = 0;
            for (int i = 0; i < n; i++) {
                int s = sc.nextInt();
                int t = sc.nextInt();
                
                dap += s * (t - f);
                f = t;
            }
            System.out.println(dap + " miles");
        }
        sc.close();
    }
}

File: 백준/Bronze/14541. Speed Limit/Speed Limit.c
#include <stdio.h>

int main() {
    while (1) {
        int n;
        scanf("%d", &n);        
        if (n == -1) {
            break;
        }

        int dap = 0, f = 0;        
        for (int i = 0; i < n; i++) {
            int s, t;
            scanf("%d %d", &s, &t);
            
            dap += s * (t - f);
            f = t;
        }        
        printf("%d miles\n", dap);
    }
    return 0;
}